### ISSUE TITLE
YouTube Videos Not Pausing Anymore on Chrome 100.0.4896.127 (Official Build) (64-bit)

### ISSUE THREAD

**Issue by thenonhacker** [james-fray/YouTube-No-Buffer#17]
# YouTube Videos Not Pausing Anymore on Chrome

## Chrome Version
100.0.4896.127 (Official Build) (64-bit)

## YouTube™ No Buffer Version
0.4.3

## Selected Settings:
✅ Stop even on playlists
✅ Pause the player instead of stopping it3
✅ Pause the player instead of stopping it if the URL has seek argument ("&t=xs")

## What I already tried
1. Update and Restart Chrome
2. Uninstall / Reinstall Extension
3. Clear Cache
4. Refresh YouTube Page to make sure that the plugin reloads on YouTube.

**Comment by james-fray** [comment]
Have you tried on a clean profile? Does it work if you uncheck "Pause the player instead of stopping it"?

**Comment by thenonhacker** [comment]
> Does it work if you uncheck "Pause the player instead of stopping it"?

I did, and still, it has no effect on the YouTube Videos.


> Have you tried on a clean profile? 

Ok, I started an Incognito Window, which is Not tied to my main Chrome profile, and then enabled this extension in Incognito Mode.

Then I went to a YouTube video.

Same bug and behavior occurred.

![image](https://user-images.githubusercontent.com/1762699/163718031-1d9e3a9c-8e91-4e0f-af4e-fc71ec3323db.png)

**Comment by james-fray** [comment]
> which is Not tied to my main Chrome profile

Not really. I want to make sure that no other extension is interrupting the worker registration. Try on a temporary clean profile instead.

Another test:
1. Right-click on a YouTube page and select the "Inspect" item.
2. Switch to the console tab.
3. Reload the page and monitor the log. 

Do you get any useful logs here?

**Comment by thenonhacker** [comment]
Tested again.

**Only Video Resumer is the plugin working in Incognito mode. So no other extension should interrupt that:**

![image](https://user-images.githubusercontent.com/1762699/164022693-8b87e4e6-40ee-486b-a49f-c3c395a8ed0b.png)


**Console logs as requested after refreshing a YouTube video:**

![image](https://user-images.githubusercontent.com/1762699/164023245-2b655350-e552-42d9-9285-ef17aece895a.png)

**Comment by thenonhacker** [comment]
The extension works if I create a new profile, and that's the only extension installed.

**Comment by thenonhacker** [comment]
UPDATE: I found the culprit. 

If I disable [YouTube Normalizer](https://chrome.google.com/webstore/detail/youtube-normalizer/ihlalbalpneokgcdiafhkjpnbekhlnmc), your extension will work again.

**Comment by james-fray** [comment]
I just tried both extensions in a clean profile and they work fine for me.

**Comment by thenonhacker** [comment]
Hi James!

You are right... I think I found the exact scenario where your plugin didn't pause / stop the video.

When I open a YouTube video on a new tab, your plugin successfully stops the video from playing (and therefore, buffering).

The problem is when I open YouTube video link with a Time Request Parameter.

**Example:** 

Plugin will successfully stop this:
https://www.youtube.com/watch?v=hJNF2_dCWkg

Plugin won't be able to pause this:
https://www.youtube.com/watch?v=hJNF2_dCWkg&t=5s

**Comment by james-fray** [comment]
The second link has a seek argument (`t=`). The extension does not stop these types of videos, since the position will be forgotten. You can change this behavior on the options page.

**Comment by hassanbenhzaine** [comment]
Hello, I have Chromium 107.0.5304.87 on GNU/Linux Debian and its working fine, try to clean and reinstall latest version that might work.
